clientv3/integration/kv_test.go

```diff
@@ -35,8 +35,8 @@ func TestKVPutError(t *testing.T) {
 	defer testutil.AfterTest(t)
 
 	var (
-		maxReqBytes = 1.5 * 1024 * 1024
-		quota       = int64(maxReqBytes * 1.2)
+		maxReqBytes = 1.5 * 1024 * 1024      // hard coded max in v3_server.go
+		quota       = int64(4 * 1024 * 1024) // the grpc max
```

Comment:
This test should have nothing to do with grpc limits. What was wrong with `maxReqBytes * 1.2`?